types/jest-retries/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "esModuleInterop": true,
```

Comment:
```suggestion
```
it's not reuqired here, only useful for non-CJS modules, which this one is AFAIK:
https://github.com/bluzi/jest-retries/blob/master/src/retry.js#L43